plugins/modules/oneview_appliance_device_snmp_v3_users.py

```diff
@@ -138,10 +138,11 @@ def __init__(self):
         self.set_resource_object(self.oneview_client.appliance_device_snmp_v3_users)
 
     def execute_module(self):
+        parameter_to_ignore = "authenticationPassphrase"
```

Comment:
if there are multiple properties, we need to ignore then it would not work. For example AuthenticationPassphrase, PrivacyPassphrase. So better to make parameter_to_ignore as list but I think its we are ok current change for now. We will see in later date if multiple property ignore required